platform_it/pom.xml

```diff
@@ -1274,6 +1315,10 @@
                                 <argument>-XX:+FlightRecorder</argument> -->
                                 <argument>-Djava.security.krb5.conf=${project.build.directory}/krb5.conf</argument>
                                 <argument>-Dlog4j.configuration=${log4j.config.uri}</argument>
+                                <!--<argument>-agentpath:/opt/jprofiler/bin/linux-x64/libjprofilerti.so=port=8851</argument>-->
```

Comment:
same here: if the intent is to say, e.g. : to run the profiler uncomment this line - then i would accept the comment and the line, but it just fills the file with extra content?